corehq/apps/reports/views.py

```diff
@@ -1710,7 +1710,7 @@ def _sorted_case_update_keys(keys):
 def _get_form_metadata_context(domain, form, timezone, support_enabled=False):
     from corehq.apps.hqwebapp.templatetags.proptable_tags import get_default_definition, get_tables_as_columns
 
-    meta = form.metadata.to_json()
+    meta = form.metadata.to_json() or {}
```

Comment:
This won't fix the error in the ticket since it is 'form.metadata' that is returning None.

You'll need to check if form.metadata is None before calling 'to_json()'

Reply:
```suggestion
    meta = form.metadata.to_json() if form.metadata else {}
```